app/models/goal.py

```diff
@@ -1,5 +1,33 @@
 from app import db
-
+from app.models.task import Task
 
 class Goal(db.Model):
+
     goal_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    tasks = db.relationship('Task', backref='goal', lazy=True)
+
+
+    def to_dict(self):
+        goal_as_dict={}
+        
+        goal_as_dict["id"] = self.goal_id
+        goal_as_dict["title"] = self.title
+        return goal_as_dict
```

Comment:
Refactoring suggestion: Consider making this an inline return in order for this method to feel "cleaner"

```python
    def to_dict(self):
        return {
            "id": self.goal_id,
            "title": self.title
        }
```